dsa-solutions/lc-solutions/0000-0099/0062-Unique-path.md

```diff
@@ -0,0 +1,134 @@
+---
+id: Unique Path
```

Comment:
replace `Unique Path` to `unique-path`

Reply:
@Ajay-Dhangar i have updated the id name , kindly merge my PR .